src/services/iiif.test.ts

```diff
@@ -1,6 +1,12 @@
-import { label } from "./presentation";
+import { getLabel } from './iiif';
```

Comment:
Is there a Hyperion helper method to do what `getLabel` is doing here?

Reply:
@adamjarling There may be through use of Traverse from the parser package. I spent some time looking at this and Traverse seems really helpful for finding **all labels** or **all canvases**. For a top level label it felt like overkill, but I do want to wrap my mind around this more and see where and when we apply Traverse.